sources/lib/frontend/typechecker.ml

```diff
@@ -52,7 +52,7 @@ module Types = struct
     { to_ty = MString.empty;
       from_labels = MString.empty }
 
-  let fresh_vars ~recursive env vars loc =
+  let fresh_vars ~recursive _ vars loc =
```

Comment:
If the argument env is not used anymore should it be better to remove it instead of _ ?